jarviscli/plugins/joke_of_day.py

```diff
@@ -0,0 +1,52 @@
+from plugin import plugin, require
+import requests
+from colorama import Fore
+from plugins.animations import SpinnerThread
+
+
+@require(network=True)
+@plugin('jokeofday')
```

Comment:
Would suggest we name this plugin just 'joke'. And rename current 'joke' plugin to 'joke chuck'.